service/history/timer_queue_standby_task_executor.go

```diff
@@ -294,7 +294,7 @@ func (t *timerQueueStandbyTaskExecutor) executeActivityRetryTimerTask(
 			return nil, nil
 		}
 
-		return newActivityRetryTimePostActionInfo(mutableState, activityInfo.TaskQueue, activityInfo.ScheduleToStartTimeout.AsDuration(), activityInfo.UseCompatibleVersion)
+		return newActivityRetryTimePostActionInfo(mutableState, activityInfo.TaskQueue, activityInfo.ScheduleToStartTimeout.AsDuration(), activityInfo)
```

Comment:
just checking: this will push activity to standby matching using the buildID from the active cluster which could be unknown to the standby cluster. and this case is fine right?

Reply:
yes, Matching does not require the build ID be known beforehand.